tests/typeConverter.test.ts

```diff
@@ -13,7 +15,7 @@ describe('TS types generation', () => {
         const result = parseObject({ schema, schemaKey: 'TypeWithId' });
 
         const expectedString = `export interface TypeWithId {
-    id: string; // format: "guid"
+\tid: string; // format: "guid"
```

Comment:
in the helper, I replace the 4 space by the tab `\t` characters.

Reply:
4 space tab is not robust enough?
I personaly always prefered space tabs instead of actual tabs...

Reply:
I guess it come down to preference .
Artem comment was to avoid change in the test result, even if it's just spacing. So I will revert that change

Reply:
okk, no worries otherwise